common/culture/cultures/wc_hyenaic.txt

```diff
@@ -6,6 +6,11 @@
 	language = language_gnoll
 	martial_custom = martial_custom_male_only
 	traditions = {
+		tradition_warrior_culture
```

Comment:
I would add/replace something with tradition_battlefield_looters since they're known for eating and looting corses after the battle.